src/content/en/feedback/file-a-bug.md

```diff
@@ -0,0 +1,107 @@
+project_path: /web/_project.yaml
+book_path: /web/feedback/_book.yaml
+
+{# wf_updated_on: 2016-10-24 #}
+{# wf_published_on: 2016-10-24 #}
+
+# How to File a Good Bug {: .page-title }
+
+Filing a good bug is easy, but does take a little work. The goal is to make
+it as easy as possible for someone to see what's broken, figure out what
+broke, and most importantly, how to fix it.
+
+Bugs that get traction tend to be easy to reproduce and easy to understand
+the expected outcome.
+
+## Are others seeing the same problem?
+
+If you’re experiencing problems, there’s a good chance other developers are too.
+Try searching for the bug on Stack Overflow. This might help you translate an
+abstract problem into a specific broken API. It might also give you a
+workaround for the short term.
+
+Once you have an idea of what the bug is, search for it on the
+[browser bug searcher](https://developer-feedback.appspot.com/web/feedback/).
+If you find an existing bug that describes the problem, it’s much more useful
+to add your support by starring, favoriting, or commenting on that bug.
```

Comment:
Commenting? You should elaborate or qualify this so that we're not encouraging spam in our bugs.